src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,46 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static BookDao bookDao;
 
+    public static void main(String[] args) {
+        bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book1 = new Book();
```

Comment:
name with numbers is bad approach

You can create constructor with all parameters for Book and create list here
```
List<Book> books = List.of(
                new Book(
```